docs/issue_tracking.md

```diff
@@ -69,7 +69,8 @@ Issues can be blocked for some different reasons:
 * Refinement - waiting for agreement that it is ready to go from Planning to
   Ready
 * Blocked - waiting for some other dependency before work on this one can
-  continus.
+  continus
+* Review - waiting on design review
```

Comment:
I feel like folks will get this and "PR" confused. Should we call this "Design Review"?

Reply:
I agree that those reasons are somewhat annoying and begrudgingly accept them.